src/Params.ml

```diff
@@ -291,7 +301,10 @@ let get_if_then_else (c : Conf.t) ~first ~last ~parens ~parens_bch
       ; cond= cond ()
       ; box_keyword_and_expr= Fn.id
       ; branch_pro
-      ; wrap_parens= wrap_if parens_bch "(@;<0 2>" ")"
+      ; wrap_parens=
+          wrap_parens_or_beginend_if c ~parens_bch ~exp_grouping_bch
+            ~opn_hint:((0, 2), (0, 2))
+            ~cls_hint:((0, 0), (0, 0))
```

Comment:
Maybe it makes sense to have opn_hint and cls_hint (the names could be improved) in the record instead of wrap_parens?

Reply:
They are used differently for parens and for begin-end. Also, there is the `indicate_multiline_delimiters` option that should stay in this module.